src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -1028,14 +1023,22 @@ private void DynamoView_Loaded(object sender, EventArgs e)
             {
                 CheckTestFlags();
             }
-            //if feature flags is not yet initalized, subscribe to the event and wait.
+            //if feature flags is not yet initialized, subscribe to the event and wait.
             else
             {
                 DynamoUtilities.DynamoFeatureFlagsManager.FlagsRetrieved += CheckTestFlags;
             }
            
         }
 
+        private void GuideFlowEvents_GuidedTourStart(GuidedTourStateEventArgs args)
```

Comment:
Is the library section width decreased? The node that needs to be placed is not clearly shown.

Reply:
200 seems to be the default width